### PR TITLE
Updated EventStreamHandler for REST API docs

### PR DESCRIPTION
**Pull request by acald-creator**
**Notes for Reviewers**

This PR updates the swagger generation, adding EventStreamHandler entry with swagger

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/helpers/swagger.yaml [hunk 1]
```diff
@@ -242,6 +242,16 @@ paths:
             summary: Handle POST request for Sharing content
             tags:
                 - ShareContent
+    /api/events/events-streamer:
+        get:
+            description: Fetches events from Remote Provider and streams it to the UI
+            operationId: idGetEventStreamer
+            responses:
+                "200":
+                    description: ""
```
Comment (leecalcote) [inline server/helpers/swagger.yaml:251]: @Revolyssup what do we have for a description here?

Comment (acald-creator) [reply server/helpers/swagger.yaml:251]: There aren't any descriptions among the other endpoints, but there is a general description anyway on the handler itself. If we need to add a description, this should be the same for the other endpoints too

Comment (leecalcote) [reply server/helpers/swagger.yaml:251]: @acald-creator you're so very much right: all of the endpoints need additional documentation. As an extensible platform, Meshery needs very well-documented APIs.  Yes, please.

File: server/handlers/events_streamer.go [hunk 2]
```diff
@@ -22,6 +22,13 @@ var (
 	flusherMap map[string]http.Flusher
 )
 
+// swagger:route GET /api/events/events-streamer EventsAPI idGetEventStreamer
+// Handle GET request for events streamer
+//
+// Fetches events from Remote Provider and streams it to the UI
+// responses:
+// 	200:
+
```
Comment (Azanul) [inline server/handlers/events_streamer.go:31]: No parameters listed and no response object definition?

**Review by Azanul — COMMENTED** [review]
.

File: server/handlers/events_streamer.go (hunk 2 above)
Comment (acald-creator) [reply server/handlers/events_streamer.go:31]: Most of the others are like this.

This is why I mentioned that if this is going to be the case here, this type of change needs to be reflected across the rest of the endpoints.

File: server/handlers/events_streamer.go [hunk 3]
```diff
@@ -22,6 +22,13 @@ var (
 	flusherMap map[string]http.Flusher
 )
 
+// swagger:route GET /api/events EventsAPI idGetEventStreamer
+// Handle GET request for events.
+// Listens for events across all of Meshery's components like adapters and server, streaming it to the UI
+// This API call never terminates and establishes a persistent keep-alive connection over which EventsResponse is sent back.
```
Comment (leecalcote) [inline server/handlers/events_streamer.go:28]:
```suggestion
// This API call never terminates and establishes a persistent keep-alive connection over which `EventsResponse`s are pushed.
```

**Comment by l5io** [comment]
🚀 Preview for commit 86e758bee965b7362ebdf9341494df213933f234 at: https://643d5ab5b5cbf51d24482f7e--meshery-docs-preview.netlify.app

**Pull request merged**